test/server/services/user.js

```diff
@@ -442,5 +442,15 @@ describe('User service tests', function() {
           done();
         }, done);
     });
+    it('should clear the call_sid for user on clearUserCallSID service', (done) => {
```

Comment:
This test is not really doing anything since the user call SID for this fixture is already null because of the tests about it. It should be set to something before it can be cleared.